radis/io/hdf5.py

```diff
@@ -209,6 +218,94 @@ def __del__(self):
             self._close_temp_batch_files()
 
     def load(
+        self,
+        fname,
+        columns,
+        lower_bound=[],
+        upper_bound=[],
+        within=[],
+        output="pandas",
+        **store_kwargs,
+    ):
+        """
+        Other Parameters
+        ----------------
+        columns: list of str
+            list of columns to load. If ``None``, returns all columns in the file.
+        output: 'pandas', 'vaex', 'jax'
+            format of the output DataFrame. If ``'jax'``, returns a dictionary of
+            jax arrays.
+        lower_bound: list of tuples [(column, lower_bound), etc.]
+            ::
+
+                lower_bound =[("wav", load_wavenum_min)]
+        upper_bound_bound: list of tuples [(column, upper_bound), etc.]
+            ::
+
+                upper_bound=[("wav", load_wavenum_max)]
+        within: list of tuples [(column, within_list), etc.]
+            ::
+
+                within=[("iso", isotope.split(","))]
+        """
+        if not lower_bound and not upper_bound and not within:
+            df = self.read(
+                fname,
+                columns,
+                **store_kwargs,
+            )
+        else:
+            df = self.read_filter(
+                fname,
+                columns,
+                lower_bound=lower_bound,
+                upper_bound=upper_bound,
+                within=within,
+                **store_kwargs,
+            )
+
+        # Load
+        # Convert to output format
+        engine = self.engine
+
+        if engine == output:
+            df = df
+        elif engine == "vaex":
+            # in vaex, column selection has to happen now
+            if columns:  # load only these columns (if they exist)
+                columns = [c for c in columns if c in df.columns]
+            if output == "pandas":
+                df = df.to_pandas_df(column_names=columns)
+            elif output == "jax":
+                if columns == None:
+                    columns = ["wav", "int", "A", "El", "gpp"]
+                out = {}
+                try:
+                    import jax as jnp
```

Comment:
This should be `import jax.numpy as jnp`.

Reply:
@erwanp Can you fix this part? Then, I will try to make a prototype of the conversion from a DataFrame to ExoJAX.

Reply:
Fixed !